File: UpmeetApp/ClientApp/src/app/interfaces/event.ts
export interface Event {
  id: number;
  title: string;
  description: string;
  eventDate: Date;
  eventLocation: string;
}



export interface JoinedEvent {
  id: number;
  eventID: number;
  userID: number;
  description: string;
  eventDate: Date;
  eventLocation: string;
}



export interface Favorite {
  id: number;
  eventID: number;
  userID: number;
}

File: UpmeetApp/Models/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace UpmeetApp.Models
{
    public class Event
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime EventDate { get; set; }
        public string EventLocation { get; set; }
    }
}
